charts/harbor/v0.0.1/templates/proxy/deployment.yaml

```diff
@@ -0,0 +1,62 @@
+{{ if .Values.globalRegistryMode }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "harbor.proxy" . }}
+  labels:
+{{ include "harbor.labels" . | indent 4 }}
+    component: proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+{{ include "harbor.matchLabels" . | indent 6 }}
+      component: proxy
+  template:
+    metadata:
+      labels:
+{{ include "harbor.labels" . | indent 8 }}
+        component: proxy
+    spec:
+      containers:
+      - name: nginx
+        image: {{ template "system_default_registry" . }}{{ .Values.proxy.image.repository }}:{{ .Values.proxy.image.tag }}
+        imagePullPolicy: {{ .Values.proxy.image.pullPolicy }}
+        resources:
+{{ toYaml .Values.proxy.resources | indent 10 }}
+        ports:
+        - containerPort: 80
+        volumeMounts:
+        - name: nginx-config
+          mountPath: /etc/nginx/conf.d
+          readOnly: true
+        - mountPath: /etc/nginx/filters.conf
+          name: filter-config
+          subPath: filters.conf
+          readOnly: true
+      volumes:
+      - name: nginx-config
+        configMap:
+          name: "{{ template "harbor.proxy" . }}"
+          items:
+            - key: default.conf
+              path: default.conf
+      - name: filter-config
+        configMap:
+          name: "{{ template "harbor.proxy" . }}"
+          items:
+          - key: filters.conf
+            path: filters.conf
+    {{- with .Values.proxy.nodeSelector }}
+      nodeSelector:
```

Comment:
Make sure all workloads don't be scheduled into Windows host. You could choose to use `node selector` or `node affinity`.